tests/test_decision_maker/test_base.py

```diff
@@ -102,7 +103,7 @@ def test_transaction_is_affordable_agent_is_buyer(self):
         )
         tx_message = TransactionMessage(
             performative=TransactionMessage.Performative.PROPOSE_FOR_SETTLEMENT,
-            skill_callback_ids=["default"],
```

Comment:
the `"default"` didn't make any sense, right? we do not have a skill called `"default"`

Reply:
Erm yes, makes no sense you are right!